pmpro-bbpress.php

```diff
@@ -60,7 +60,7 @@ function pmprobb_init() {
  * @since 1.7
  */
 function pmprobb_filter_forum_search_results( $query ) {
-	if ( apply_filters( 'pmprobb_filter_topic_queries', true ) && function_exists('bbp_is_search_results') && bbp_is_search_results() ) {
+	if ( ! is_admin() && apply_filters( 'pmprobb_filter_topic_queries', true ) && function_exists('bbp_is_search_results') && bbp_is_search_results() ) {

```

Comment:
Don't know if it matters, nor if it's still true, but note that `is_admin()` is false when in the backend block editor: https://core.trac.wordpress.org/ticket/47394